spec/models/ems_event_spec.rb

```diff
@@ -196,6 +197,27 @@
           expect(new_event.availability_zone_id).to eq @availability_zone.id
         end
       end
+
```

Comment:
".add" context is nested under "with availability zones"
Not sure if it's less confusing to add the new context here or create another top level ".add" context